bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| externalSensorKwlVoc5            | vocControl      | External VOC sensor 5                                            | Number:Dimensionless     | ppm  | -0.0 - 9998.9                                                                                        |
+| externalSensorKwlVoc6            | vocControl      | External VOC sensor 6                                            | Number:Dimensionless     | ppm  | -0.0 - 9998.9                                                                                        |
+| externalSensorKwlVoc7            | vocControl      | External VOC sensor 7                                            | Number:Dimensionless     | ppm  | -0.0 - 9998.9                                                                                        |
+| externalSensorKwlVoc8            | vocControl      | External VOC sensor 8                                            | Number:Dimensionless     | ppm  | -0.0 - 9998.9                                                                                        |
+| nhzDuctSensor                    | general         | After-heater intake temperature in °C                            | Number:Temperature       | °C   | -27.0 - 9998.9                                                                                       |
+| weekProfileNhz                   | profiles        | Week profile after-heater                                        | Number                   |      | 0 = standard 1, 1 = standard 2, 2 = fixed value, 3 = individual 1, 4 = individual 2, 5 = NA, 6 = off |
+| serNo                            | general         | Serial number                                                    | String                   |      |                                                                                                      |
+| prodCode                         | general         | Production Code                                                  | String                   |      |                                                                                                      |
+| supplyAirRpm                     | general         | Supply air fan RPM                                               | Number                   |      | 0 - 9999                                                                                             |
+| extractAirRpm                    | general         | Extract air fan RPM                                              | Number                   |      | 0 - 9999                                                                                             |
+| logout                           | general         | Logout                                                           | Switch                   |      | ON = logout                                                                                          |

```

Comment:
What's the purpose of this Channel?

Reply:
It's also part of the device's Modbus protocol specification. As far as I can tell this it what the devices integrated web based config app uses to log out the user from the web page. I'm actually not sure if it makes a lot of sense in the binding.
Probably it can be removed...